pkgs/development/python-modules/arelle/tests.patch

```diff
@@ -0,0 +1,32 @@
+diff --git c/arelle/CntlrProfiler.py i/arelle/CntlrProfiler.py
+index a64bb1b..dc0299d 100644
+--- c/arelle/CntlrProfiler.py
++++ i/arelle/CntlrProfiler.py
+@@ -1,5 +1,6 @@
+ 
+-import Cntlr, ModelManager, FileSource, time
++from arelle import Cntlr, ModelManager, FileSource
++import time
+ from optparse import OptionParser
+ import cProfile
+ import gettext
+diff --git c/arelle/ValidateFilingText.py i/arelle/ValidateFilingText.py
+index 12dbbbb..c0e98c3 100644
+--- c/arelle/ValidateFilingText.py
++++ i/arelle/ValidateFilingText.py
+@@ -16,7 +16,7 @@ XMLdeclaration = re.compile(r"<\?xml.*\?>", re.DOTALL)
+ XMLpattern = re.compile(r".*(<|&lt;|&#x3C;|&#60;)[A-Za-z_]+[A-Za-z0-9_:]*[^>]*(/>|>|&gt;|/&gt;).*", re.DOTALL)
+ CDATApattern = re.compile(r"<!\[CDATA\[(.+)\]\]")
+ #EFM table 5-1 and all &xxx; patterns
+-docCheckPattern = re.compile(r"&\w+;|[^0-9A-Za-z`~!@#$%&\*\(\)\.\-+ \[\]\{\}\|\\:;\"'<>,_?/=\t\n\r\m\f]") # won't match &#nnn;
++docCheckPattern = re.compile(r"&\w+;|[^0-9A-Za-z`~!@#$%&\*\(\)\.\-+ \[\]\{\}\|\\:;\"'<>,_?/=\t\n\r\f]") # won't match &#nnn;
```

Comment:
I had to look this up. The table mentioned in the comment ( EFM table 5-1 in https://www.sec.gov/info/edgar/edgarfm-vol2-v25.pdf ) does not seem to include anything that warrants the `\m`. Also it's hard to find a meaning for `\m` in a python regex. Probably ok. I'll check with the author asynchronously.